src/compat.rs

```diff
@@ -0,0 +1,187 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Are these kinks that will get worked out before stability?

Reply:
No. These are fundamental differences between futures 0.1 and 0.3. They can be removed once we migrate the upstream (grpc) code to futures 0.3.

Reply:
Ok. 👍